### DISCUSSION TITLE
[Script request] ubuntu business desktop lxc

### DISCUSSION THREAD

**Discussion by amlucent** [community-scripts/ProxmoxVE#573] [Request script]
### Application Name

ubuntu business desktop lxc

### Website

https://sourceforge.net/projects/ubuntu-business-desktop-lxc/

### Description

ubuntu business desktop lxc is a Virtual Desktop LXC-Template witch can be used for a company or private. It provides a Ubuntu virtual desktop GUI experience in an LXC for as many users as you want.  It's an adapted Ubuntu with x2go-Terminalserver and Webmincontrol. It's easy to control the system with webmin. 

This is a ubuntu based LXC with a graphics enabled desktop.  It can be used for virtual desktop infrastructure and terminal server purposes.  Its fully ready and configured for a proxmox installation and full installation requirements and instructions can be found here https://git.osit.cc/public-projects/ubuntu-lxc-terminalserver-project.

I am using this on my proxmox server, I think its cool, I think the community would benefit from the easy installation and setup script for this.  One benefit of this solution being a LXC means you can use a graphics card or other hardware in the container without IOMMU passthrough.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Creating a script for running graphical interfaces like Ubuntu Desktop in LXC is too much effort for the limited demand. Configuring Wayland/X11 and ensuring compatibility (e.g., with Proxmox) requires significant manual adjustments, making it impractical for most users. Alternative solutions like VMs or remote desktop tools are more suitable.

**Reply by amlucent** [reply]
Have you followed the link?  All the things you mention are done for you with with this prebuilt container.   Im suggesting to create a easy deployment script for this.

**Comment by jsapede** [comment]
would be nice to have remote desktop using guacamole / RDP as novnc is unavailable behind FW at work

**Comment by jsapede** [comment]
thanks for the link was looking for a working desktop ubuntu since ages. 
simply added xrdp and works out of the box with guacamole
